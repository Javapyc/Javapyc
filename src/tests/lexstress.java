.||6969726322 ydE0a6k4fq2l75u[!!=
PM23117Ff7N
/*= / . 9592 6639064074123701 54420 / 
] <= 18992861560 2591316 WaU6Jfj254N1 ! public 
st284m8Z { { || T72t8262x549x ! 81868941 
*/=!/*<= false d51D08 q0h4 != public { 
/ { 927502336460919 } || false 0 
SP7 true void System.out.println 487013 ) c27b66SLK555 
*/,[390197620573686]P31I41h8GMcdgx1 while E601C03Q44Ug3CiI false public	+else||//] [ ! { . 567 class 
mO9iJ} extends
484764481159 574160]-!=main	209*970062348426625 true
[H98a3S789M1huh<||[ 2832082979582+jy10M185z 131280 171312282650787!5328233577 239820]in3m772C(true/*( ) != 383840939582875 w4M1de73Tg82fm ; 6285001164 
) , < , sr6F6456WRW24 { == 
&& == 9542151 2049 90812909 35272 ! 
*/ NAozH75u 9226591/{oH!//&& 8569357948740852 . ; != , == 
 >=
91612967363740//901849976615110 ) + } h07e8y1Kg8U else Lp3fRD50t5jUB59b 
 G69JqaX5b2w1OA 0!=
Wf9 y7mZ9&&//{ 18586619 this 482073685063 k7 - 3743919268881494 
Cx7y2f76ZI0Xx3 44157 nEXKE86R)D5=*44533>
=m219y28U5Q-!3168652564699//; 917217124407217 c04PStp49N02 6847421421758178 = boolean lck0r 
.]a54K516599I5x15==String/*|| + || ( this 3242466657399806 N1a7uX0v8U 
438059687730 8415696231735875 { != main void ; 
3386272347 int , 66 class = , 
*/ BoM5gZRPIMUl
