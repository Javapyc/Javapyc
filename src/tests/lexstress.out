Delimiter, .
Operator, ||
Integer, 6969726322
ID, ydE0a6k4fq2l75u
Delimiter, [
Operator, !
Operator, !=
ID, PM23117Ff7N
Delimiter, =
Operator, !
Delimiter, ,
Delimiter, [
Integer, 390197620573686
Delimiter, ]
ID, P31I41h8GMcdgx1
ReservedWord, while
ID, E601C03Q44Ug3CiI
ReservedWord, false
ReservedWord, public
Operator, +
ReservedWord, else
Operator, ||
ID, mO9iJ
Delimiter, }
ReservedWord, extends
Integer, 484764481159
Integer, 574160
Delimiter, ]
Operator, -
Operator, !=
ReservedWord, main
Integer, 209
Operator, *
Integer, 970062348426625
ReservedWord, true
Delimiter, [
ID, H98a3S789M1huh
Operator, <
Operator, ||
Delimiter, [
Integer, 2832082979582
Operator, +
ID, jy10M185z
Integer, 131280
Integer, 171312282650787
Operator, !
Integer, 5328233577
Integer, 239820
Delimiter, ]
ID, in3m772C
Delimiter, (
ReservedWord, true
ID, NAozH75u
Integer, 9226591
Operator, /
Delimiter, {
ID, oH
Operator, !
Operator, >=
Integer, 91612967363740
ID, G69JqaX5b2w1OA
Integer, 0
Operator, !=
ID, Wf9
ID, y7mZ9
Operator, &&
ID, Cx7y2f76ZI0Xx3
Integer, 44157
ID, nEXKE86R
Delimiter, )
ID, D5
Delimiter, =
Operator, *
Integer, 44533
Operator, >
Delimiter, =
ID, m219y28U5Q
Operator, -
Operator, !
Integer, 3168652564699
Delimiter, .
Delimiter, ]
ID, a54K516599I5x15
Operator, ==
ReservedWord, String
ID, BoM5gZRPIMUl
